source/common/http/http1/codec_impl_legacy.h

```diff
@@ -205,6 +205,7 @@ class ConnectionImpl : public virtual Connection, protected Logger::Loggable<Log
   }
   uint32_t bufferLimit() { return connection_.bufferLimit(); }
   virtual bool supportsHttp10() { return false; }
+  virtual bool allowChunkedLength() { return false; }
```

Comment:
This part needs to be synced up. (Sorry for the trouble here while we converge this).